mmv1/third_party/terraform/services/dataproc/resource_dataproc_cluster.go

```diff
@@ -1227,6 +1228,34 @@ func ResourceDataprocCluster() *schema.Resource {
 														},
 													},
 												},
+												"provisioning_model_mix": {
```

Comment:
Do we want to add `ForceNew: true,` as it is not updatable?

Reply:
Yes, seems like I missed it for `provisioning_model_mix` and `standard_capacity_base`.